inductiva/utils/templates.py

```diff
@@ -17,3 +17,30 @@ def replace_params_in_template(
     template = environment.get_template(template_filename)
     stream = template.stream(**params)
     stream.dump(output_file_path)
+
+
+def batch_replace_params_in_template(
+    templates_dir: str,
+    template_filename_paths: List[str],
+    params: Dict,
+    output_filename_paths: List[str],
```

Comment:
Maybe we could add a default for this, wdyt?

Reply:
But a default would need to consider the number of `template_filename_paths`. Given that there is an output_filename_path for each template_filename_path.

Reply:
Sorry. I didn't mean a default value in the function definition, but rather a default value that we assign dynamically in case the function argument is None. Couldn't it be something as simple as removing the `.jinja` part of each filename? Wdyt?

Reply:
The template filenames are a bit more explicit, e.g., "0/include/initialConditions_template.openfoam.jinja" representing the fact that it is a template and uses openfoam. 

So I think it is not so simple to provide a default value to output_filename_paths.